### ISSUE TITLE
unable to identify undefined values in class

### ISSUE THREAD

**Issue by gesslar** [fluffos/fluffos#903]
```c
class L {
    string m ;
    int n ;
    float o ;
}

void runit()
{
    class L l = new(class L) ;
    int x = 3 ;

    printf("l: %O\n", l) ;
    printf("nullp(l->m): %O\n", nullp(l->m)) ;
    printf("undefinedp(l->n): %O\n", undefinedp(l->n)) ;
    printf("typeof(l->o): %O\n", typeof(l->o)) ;

    while(x--)
    {
        printf("%O",
            sprintf("MEMBER #: %O\n", x) +
            sprintf(sprintf("VALUE: %O\n", fetch_class_member(l, x))) +
            sprintf(sprintf("TYPEOF: %O\n", typeof(fetch_class_member(l, x)))) +
            sprintf(sprintf("NULLP: %O\n", nullp(fetch_class_member(l, x)))) +
            sprintf(sprintf("UNDEFINEDP: %O\n", undefinedp(fetch_class_member(l, x))))
        ) ;
    }
}

```
results:
```
l: CLASS( 3 elements
  0,
  0,
  0
 )
nullp(l->m): 0
undefinedp(l->n): 0)
typeof(l->o): "int"
"MEMBER #: 2
VALUE: 0
TYPEOF: "int"
NULLP: 0
UNDEFINEDP: 0
""MEMBER #: 1
VALUE: 0
TYPEOF: "int"
NULLP: 0
UNDEFINEDP: 0
""MEMBER #: 0
VALUE: 0
TYPEOF: "int"
NULLP: 0
UNDEFINEDP: 0
"
```

**Comment by thefallentree** [comment]
ok: this can be fixed.   do we want to initialize all type with undefined though?   I think for int and float it should just be 0,  but string it might need to be undefined

**Comment by gesslar** [comment]
I think everything should be initialized as undefined. Since we use 1 and 0
to also mean "boolean", being undefined should show that we've not set that
value.

Brian M. Workman
***@***.***

**Comment by gesslar** [comment]
Also, it would be more consistent and expected behaviour if everything were treated the same.

**Comment by gesslar** [comment]
This issue is closed as a result of #911
